core/new-gui/src/app/workspace/component/workflow-editor/workflow-editor.component.ts

```diff
@@ -141,6 +142,29 @@ export class WorkflowEditorComponent implements AfterViewInit {
       ));
   }
 
+  private handleOperatorSuggestionHighlightEvent(): void {
+    const highlightOptions = {
+      name: 'stroke',
+      options: {
+        attrs: {
+          'stroke-width': 5,
+          stroke: '#551A8B70'
+        }
+      }
+    };
```

Comment:
Move this to a static function since it is always the same. Something like `getHighlightOptionsForRecommendation` that returns an `object` type.